packages/containerd/Cargo.toml

```diff
@@ -12,8 +12,8 @@ path = "pkg.rs"
 releases-url = "https://github.com/containerd/containerd/releases"
 
 [[package.metadata.build-package.external-files]]
-url = "https://github.com/containerd/containerd/archive/v1.6.15/containerd-1.6.15.tar.gz"
-sha512 = "5e0192d7fcffee98b5c355ab2330520dc0c64bc4c5c48111de0f84f0273707857bb6207e98c382498ea1b0448fcf7a05626113e4cffdb5cd8cbd9549346780c8"
+url = "https://github.com/containerd/containerd/archive/v1.6.19/containerd-1.6.19.tar.gz"
```

Comment:
👀

Reply:
Upgraded `host-ctr` to use `v1.6.19` containerd code libraries: https://github.com/bottlerocket-os/bottlerocket/pull/2826